installer-resources/windows/lantern.nsi

```diff
@@ -83,6 +83,10 @@ Section
     # This is a bad registry entry created by old Lantern versions.
     DeleteRegKey HKCU "Software\Microsoft\Windows\CurrentVersion\Run\value"
 
+    # Add a registry key to set -clear-proxy-settings. See https://github.com/getlantern/lantern/issues/2776
```

Comment:
Setting this registry key is a task of the installer. By default Lantern will only clear the system's proxy settings and quit unless it is asked to run at Windows startup.